doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
```

Comment:
`Data collection by The Things Stack CLI telemetry`.

Reply:
changed it to `Telemetry data collection by The Things Stack' CLI`, let me know what you think

Reply:
Yup, that's good as well.